google/ads/google_ads/interceptors/interceptor.py

```diff
@@ -200,20 +196,19 @@ def _get_error_from_response(self, response):
                 # library-specific Error type for easy handling. These errors
                 # originate from the Google Ads API and are often caused by
                 # invalid requests.
-                self._exception = GoogleAdsException(
+                return GoogleAdsException(
                     response_exception, response, google_ads_failure,
                     request_id)
             else:
                 # Raise the original exception if not a GoogleAdsFailure. This
```

Comment:
nit: we are returning the exception, not raising/throwing it.

Reply:
Fixed.